_posts/2016-11-09-making-a-small-object.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: Making a Small Object
+excerpt: The thought process behind creating a simple object to help deal with complex ActiveRecord ORDER BY queries with a strong focus on the Single Responsibility Principal.
+published: true
+categories: SQL Ruby ActiveRecord
+---
+
+I get excited about small objects when I program. I often open up files that are hundreds of lines long with mixed responsibilities and lots of complexity.
```

Comment:
For this introduction I think you could talk more about *why* you're excited by small objects.  There's an implication that small objects are easier to work with than objects which are hundreds of lines of code, but it would be even more effective to spell that advantage out explicitly. Spending more time talking about the pain of large objects and the pleasure of small objects (lol) will probably generate more excitement in the reader.